components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -267,10 +267,11 @@ tweaking their $case/namelist_scream.xml file.
     ${DIN_LOC_ROOT}/atm/scream/init/map_ne30_to_ne4_mono_20220502.nc,
     <!-- ${DIN_LOC_ROOT}/atm/scream/init/map_ne30np4_to_ne120np4_mono_20220502.nc, -->
     <!-- ${DIN_LOC_ROOT}/atm/scream/init/map_ne30np4_to_ne512np4_mono_20220506.nc, -->
-    <!-- ${DIN_LOC_ROOT}/atm/scream/init/screami_ne512np4L128_20220505.nc, -->
-    <!-- ${DIN_LOC_ROOT}/atm/scream/init/screami_ne120np4L72_20220503.nc, -->
-    ${DIN_LOC_ROOT}/atm/scream/init/screami_ne30np4L72_20220503.nc,
-    ${DIN_LOC_ROOT}/atm/scream/init/init_ne4np4.nc
+    ${DIN_LOC_ROOT}/atm/scream/init/screami_ne30np4L72_20220524.nc,
+    ${DIN_LOC_ROOT}/atm/scream/init/screami_ne4np4L72_20220524.nc,
+    ${DIN_LOC_ROOT}/atm/scream/init/screami_ne120np4L72_20220524.nc,
+    ${DIN_LOC_ROOT}/atm/scream/init/screami_ne512np4L128_20220525.nc,
+    scream-aquaplanet_init_ne4np4_L72_20220524.nc
```

Comment:
isn't this last line missing DIN_LOC_ROOT stuff?

Reply:
yes, I'll fix that.  thanks for catching it.